### PR TITLE
[Complete] - Solution - Phase2 - Day2

### PR DESCRIPTION
**Pull request by haricharanbole**
Solved telephone-check problem. The function uses a couple of regular expressions to test the telephone number and returns true if the parameter(telephone number) is according to specification otherwise returns false. I've taken two individual regular expressions for better readability and maintainability of expressions. I've used test api of RegExp. I've also included edge cases like giving multiple valid phone numbers in a string. 

P.S: I've given proper commit message. But, I missed giving commit description. The above description covers the same.

### CODE REVIEW

File: js-exercises/telephone-check/telephoneCheck.test.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { telephoneCheck } from "./telephoneCheck";
+
+describe("telephoneCheck", () => {
+  test("should return correct boolean", () => {
```
Comment (ArfatSalman) [inline js-exercises/telephone-check/telephoneCheck.test.js:4]: Both regexes fail on this input: `1  (555) 555-5555`. 
Can you take this into account?

Comment (haricharanbole) [reply js-exercises/telephone-check/telephoneCheck.test.js:4]: This is the first check in test suit. All, following tests are passing for me. Can you please confirm this `1 (555) 555-5555` again?
